### ISSUE TITLE
keyboard.toggleKey not woks as expected

### ISSUE THREAD

**Issue by ch3rn1k** [Krombik/keysender#7]
Can you explain how `keyboard.toggleKey` should work?
I expected that after using `keyboard.toggleKey(['ctrl', 'v'], true)` it should *hold* an 'Ctrl+V' key and `keyboard.toggleKey(['ctrl', 'v'], false)` should *release* it.
But I'm getting single paste as `keyboard.sendKey(['ctrl', 'v'])`

Am I doing something wrong?

**Comment by ch3rn1k** [comment]
Sample code:

```js
const win = new Hardware();|

desktop.keyboard.toggleKey(['ctrl', 'v'], true);
setTimeout(() => desktop.keyboard.toggleKey(['ctrl', 'v'], false));
```

Expected it should hold `Ctrl+V` and it will paste data in some input

**Comment by Krombik** [comment]
Toggle works as expected (you can open [keyboardchecker](https://keyboardchecker.com) and check it), but I found a bug with the `false` toggle (does not release the last key and releases the first key 2 times) and fixed it in the last version (2.0.4). If you want to paste a copy within 2 seconds, you can simply call the sendKey method in a loop.
